lib/github_api/params_hash.rb

```diff
@@ -9,7 +9,7 @@ class ParamsHash < DelegateClass(Hash)
     include Normalizer
     include MimeType
 
-    REQUEST_PARAMS = [:accept, :media, :data, :raw, :content_type, :headers]
+    REQUEST_PARAMS = [:accept, :media, :data, :raw, :content_type, :headers, :options]
```

Comment:
@piotrmurach Yes, I saw that method. But my problem is, if I remove `:options` from that list above, I get a request that looks like this:

`https://api.github.com/projects/1002604?options%5Bheaders%5D%5BAccept%5D=application/vnd.github.inertia-preview%2Bjson&options%5Braw%5D=false`